crates/core/app/src/app/mod.rs

```diff
@@ -51,6 +51,15 @@ pub mod state_key;
 /// The inter-block state being written to by the application.
 type InterBlockState = Arc<StateDelta<Snapshot>>;
 
+/// The maximum size of a CometBFT block payload (1MB)
+const MAX_BLOCK_TXS_PAYLOAD_BYTES: usize = 1024 * 1024;
+
+/// The maximum size of a single individual transaction (30KB).
+const MAX_TRANSACTION_SIZE_BYTES: usize = 30 * 1024;
```

Comment:
Did we ever choose these in a principled way? Should they be parameters rather than hardcoded constants?

Reply:
This was picked with three things in mind:
- large enough to allow JIT liquidity for multiple pairs
- large enough to allow a ranged collection of positions
- large enough to allow multiple IBC client updates
- small enough to allow many of those actions to be performed in a single block even in the extreme case where each transaction is one of those things

Making them parameters make sense, I didn't do it because it would require more work with `penumbra-governance` and hooking them into https://docs.rs/tendermint/latest/tendermint/consensus/params/struct.Params.html